tests/integration/api/test_vector_aggregation_operations.py

```diff
@@ -0,0 +1,122 @@
+"""
+Test vector aggregation operations module
+"""
+from typing import List
+
+import json
+import os
+
+import numpy as np
+import pandas as pd
+import pytest
+import pytest_asyncio
+
+from featurebyte.api.aggregator.vector_validator import VECTOR_AGGREGATE_SUPPORTED_FUNCTIONS
+from featurebyte.enum import AggFunc
+from featurebyte.query_graph.model.feature_job_setting import FeatureJobSetting
+from tests.integration.api.feature_preview_utils import (
+    convert_preview_param_dict_to_feature_preview_resp,
+)
+
+
+@pytest.fixture(name="data_with_array", scope="session")
+def data_with_array_fixture():
+    """
+    Simulated data with an array column
+    """
+    df = pd.read_csv(os.path.join(os.path.dirname(__file__), "fixtures", "vector_data.csv"))
+    # manually add the vectors on since it's easier than shoving it into a csv
+    df["VECTOR_VALUE"] = [[3, 1, 3], [1, 3, 1], [1, 5, 9], [4, 8, 6], [7, 2, 3]]
```

Comment:
Later would be good to introduce missing value as 1) elements within the vectors and 2) the vector itself to check the handling.

Reply:
yeap! will also be adding a couple more tests for different types (int vs. floats etc.)